mule-extensions-api/src/test/java/org/mule/runtime/extension/internal/loader/TransactionalDeclarationEnricherTestCase.java

```diff
@@ -149,6 +161,21 @@ public void enrichExistingTransactionalActionParameterIfExist() throws Exception
     assertThat(transactionParameter.getModelProperty(NullModelProperty.class), is(of(nullModelProperty)));
   }
 
+  @Test
+  public void throwExceptionWhenParametersOfDifferentApisArePresent() throws Exception {
```

Comment:
Nice to have this test. Is it worth to also add tests to check the other changes done to the TransactionalDeclarationEnricher, i.e. whether the extensions-api or the the sdk api annotation is present, default tx parameter value changed to sdk value, etc?

Reply:
See above that I changed the import from extensions.api to sdk.api that should cover the cases you mention.

Reply:
oh I'm missing the one where if the old api parameter is present, it preserves that type. I'll add that one.